src/main/scala/scalismo/faces/image/pyramid/GaussPyramid.scala

```diff
@@ -64,14 +59,26 @@ object GaussPyramid {
   /**
     * Standard reduce operation dividing the image size along each dimension by 2.
     */
-  def reduce[A: ClassTag](implicit ops: ColorSpaceOperations[A]) = new ImageFilter[A, A] {
+  def reduce[A: ClassTag](implicit ops: ColorSpaceOperations[A]) = reduceScaled[A](0.5)
+
+  /**
+    * Reduces image according to supplied scaling factor.
+    * @param scale 1.0 > scale >= 0
+    * @param ops
+    */
+  def reduceScaled[A: ClassTag](scale: Double)(implicit ops: ColorSpaceOperations[A]) = new ImageFilter[A, A] {
+    require( scale < 1.0 && scale >=0, "scale must be on [0,1.0)" )
+
     import ColorSpaceOperations.implicits._
+
+    val invScale = 1.0/scale
+
     override def filter(img: PixelImage[A]): PixelImage[A] = {
-      val w = img.width / 2
-      val h = img.height / 2
-      val filteredImage = img.withAccessMode(AccessMode.MirroredPositionFunctional((a:A, b:A)=>2*:a-b)).filter(GaussPyramid.filter)
+      val w = (img.width * scale).toInt
+      val h = (img.height * scale).toInt
+      val filteredImage = img.withAccessMode(AccessMode.MirroredPositionFunctional((a: A, b: A) => 2 *: a - b)).filter(GaussPyramid.filter)
       val interpolatedImage = filteredImage.interpolate
-      PixelImage[A](w, h, (x: Int, y: Int) => filteredImage(x * 2 + 1, y * 2 + 1))
+      PixelImage[A](w, h, (x: Int, y: Int) => filteredImage((x * invScale).toInt + 1, (y * invScale).toInt + 1))
```

Comment:
If the scale factor is now free, this transformation is a bit dangerous. It actually works for integer relations 1:2,3, .... However, fractional access requires interpolation and is probably overkill because the image is heavily smoothed anyway. But it is something to keep in mind.